org.eclipse.tm4e.ui/META-INF/MANIFEST.MF

```diff
@@ -15,7 +15,8 @@ Require-Bundle: org.eclipse.tm4e.core,
  org.eclipse.ui.ide;resolution:=optional,
  com.google.gson,
  org.eclipse.e4.ui.css.swt.theme,
- org.eclipse.core.expressions
+ org.eclipse.core.expressions,
+ org.eclipse.ui.genericeditor;bundle-version="1.1.0";resolution:=optional
```

Comment:
Not really. However, I believe if we include that patch as it, it would fail with generic editor 1.0.0. That's why I preferred to explicit the version of the dependency here, but still making it optional so it's not blocking on earlier versions.